__tests__/app.test.js

```diff
@@ -335,4 +335,27 @@ describe("/api", () => {
         });
     });
   });
+  describe("GET /api/users", () => {
+    test("GET 200: Should respond with an array of all user object with the correct properties", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({ body }) => {
+          const {users} = body;
+          expect(users.length).toBe(4);
+          expect(Array.isArray(users)).toBe(true);
+          users.forEach((user) => {
+            expect(user).toHaveProperty("username");
+            expect(user).toHaveProperty("name");
+            expect(user).toHaveProperty("avatar_url");
+          });
+        });
+    });
```

Comment:
Yes good work here, checking the array length is good to see and that the object has all the properties expected - well done. Consider checking the data type of each property also, i.e. is username a string? etc.